src/modules/hosts/host-list/host-list.js

```diff
@@ -15,7 +15,7 @@
         });
 
     /*@ngInject*/
-    function hostController($scope, $filter, $rootScope, $state, $interval, $uibModal, utils, config, nodeStore, clusterStore) {
+    function hostController($scope, $filter, $rootScope, $state, $interval, $uibModal, utils, config, nodeStore, clusterStore, Notifications) {
```

Comment:
why we injected `Notifications`? I don't see any use of it in this PR?

Reply:
@cloudbehl, `Notifications` is used in other places on the view, and might have left out earlier to be added.

Reply:
@a2batic ok, makes sense! But let's follow the process.

Let the program take call on this and if they feel it's critical then we will fix it then will do it. what you say?

Can you create a another Issue for this bug?

Reply:
Let's not take in this change now. revert back the notification related change, raise a git issue for the same. I am not very keen to take this in for 3.4.0. Let's fix this in the next update.